client/src/pages/Dashboard.jsx

```diff
@@ -1,31 +1,58 @@
 import React from "react";
+import axios from "axios";
+import {Redirect, useHistory } from "react-router-dom";
 import Button from "@material-ui/core/Button";
 import CssBaseline from "@material-ui/core/CssBaseline";
 import TextField from "@material-ui/core/TextField";
 import Paper from "@material-ui/core/Paper";
-import { useHistory } from "react-router-dom";
 
 export default function Dashboard() {
   const history = useHistory();
+  const [user, setUser] = React.useState(null)
+  const [loading, setLoading] = React.useState(true)
+
+  const getUser = async () => {
+    try {
+      const {data} = await axios.get('/users/auth');
+      localStorage.setItem("userInfo",JSON.stringify(data.user));
+    } catch (error) {
+      console.log(error.message);
```

Comment:
Let's not console.log on the front end at all. If it's an error that's important, we should display in a snackbar or something for the user to see